LibertyDSNP/example-client
# Feature/feed loading system 177893640

Purpose
---------------
As a developer I want a system that handles loading feed data from the sdk and supply both the new and old content as needed.


Solution
---------------
Create the required code to retrieve feed data and content and supply it to our redux store and components as needed.


Change summary
---------------
* Added new and old feed loading SDK API endpoints
* Added content loading server API endpoints
* Created mock endpoints for server and sdk as placeholders until functional
* Modified feed slice of redux store to handle feed data better
* Added unrefined feed loading implementation to components
* Fixed tests to include redux store

Steps to Verify
----------------
1. Start up example client
2. Feed should load
3. Content should appear for each post
4. Replies should be attached to their post
5. After a small delay "new content button" appears
6. Clicking button loads new feed posts
7. At any point check redux store for proper storage of data

Additional details / screenshot
----------------
![image](https://user-images.githubusercontent.com/4521469/117719852-87bc9200-b1a3-11eb-8742-e64755fc4a9a.png)
![image](https://user-images.githubusercontent.com/4521469/117719876-90ad6380-b1a3-11eb-8e8c-93bcccea0943.png)
![image](https://user-images.githubusercontent.com/4521469/117719911-9b67f880-b1a3-11eb-8a44-3eda7c64e7a6.png)
![image](https://user-images.githubusercontent.com/4521469/117719936-a4f16080-b1a3-11eb-82cf-1720a08add41.png)
